pkg/glusterblock/config/config.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+// Config struct fills the parameters of request or user input
+type Config struct {
```

Comment:
I think this can be moved out of gluster-block and keep this as common for both file and loopback to avoid code duplication

Reply:
The config of the both are little different and might get even diverse going forward.